kyuubi-server/src/main/resources/META-INF/services/org.apache.kyuubi.credentials.HadoopDelegationTokenProvider

```diff
@@ -17,3 +17,5 @@
 
 org.apache.kyuubi.credentials.HadoopFsDelegationTokenProvider
 org.apache.kyuubi.credentials.HiveDelegationTokenProvider
+org.apache.kyuubi.credentials.YarnRMDelegationTokenProvider
```

Comment:
What is this `org.apache.kyuubi.credentials.YarnRMDelegationTokenProvider`

Reply:
fixed, this line is irrelevant to this patch.